vuu-ui/packages/vuu-table-extras/src/column-formatting-settings/ColumnFormattingPanel.tsx

```diff
@@ -123,3 +115,20 @@ export const ColumnFormattingPanel = ({
     </div>
   );
 };
+
+function formattingSettingsByColType(props: FormattingSettingsProps) {
+  const { column } = props;
+
+  if (isDateTimeColumn(column)) {
+    return <DateTimeFormattingSettings {...props} column={column} />;
```

Comment:
This is good but there's still a gap in functionality. We're showing the data/time config settings if the column has already been configured as a 
date/time type. But the config settings panel should be where we can actually assign date/time as the type to a column of type `long`

I see two possibilities

1) we populate the renderer dropdown with an alternative to the 'default long renderer', we add a 'default date/time' renderer. This would assign the date/time type to a `long` column.
technically we're not actually implementing a renderer, just a formatter.

2) We add an additional UI item, to allow toggle on a long column, so user can pick between `number` or `date/time` When thet switch to date/time, we render the new settings

wdyt ?

Reply:
Yeah that's true. To me the second option sounds better (though the first one seems easier to implement, second might need passing down a separate action handler for handling type change).

Mixing up renderers with formatters could end up confusing devs, and it could also be harder for the users to navigate in case we have many renderers.